agent/netconfig.go

```diff
@@ -22,12 +22,24 @@ import (
 	"net"
 )
 
+const (
+	fullIsolationOption = "full_isolation"
+)
+
+// NetworkRequest specifies messages sent to the
+// agent containing information on how to configure network
+// on its host.
+type NetworkRequest struct {
```

Comment:
I don't know how i feel about `NetworkRequest`, is it necessarily coming from network ?